include/odp/api/spec/schedule.h

```diff
@@ -45,6 +46,53 @@ extern "C" {
  * Maximum schedule group name length in chars including null char
  */
 
+/**
+ * Schedule configuration options
+ */
+typedef struct odp_schedule_config_t {
+
+	/* Scheduler flow aware
+	 * Application can configure the scheduler to be flow aware or unaware.
+	 * When the scheduler is configured as flow aware the ORDERED and
+	 * ATOMIC schedule queue synchronization by the scheduler is performed
+	 * per flow within each queue.
+	 */
+	odp_support_t flow_aware;
+
+	/* Number of parallel flows per queue to be supported
+	 * Only valid when scheduler is configured as flow_aware
+	 *
+	 * Flows are lightweight entity and packets can be assigned to specific
+	 * flow by the application using odp_packet_flow_hash_set() before
+	 * enqueuing the packet into the scheduler.
+	 * This value is valid only when scheduler is configured in flow aware
+	 * mode.
+	 * Depening on the implementation this number might be rounded-off to
+	 * nearest supported value (e.g power of 2)
+	 * This number should be less than maximum flow supported by the
+	 * implementation.
+	 * @see odp_schedule_capability_t
+	 */
+	uint32_t flow_count;
+
+	/* Maximum number of queues to be supported
+	 * Application configures the maximum number of queues to be supported
+	 * by the implementation. This is a combination of all queue types.
+	 * @see odp_queue_capability_t
+	 */
+	uint32_t queue_count;
```

Comment:
" Maximum number of scheduled queues used

 Application will create or request other ODP APIs to create at most this many scheduled queues. See odp_queue_capability_t for scheduled queue capabilities.
"

May be queue size is also needed.
uint32_t queue_count;
uint32_t queue_size;

Reply:
queue size is already available in odp_queue_capability_t. We can add it here but it might be redundant.